index.bs

```diff
@@ -707,6 +707,7 @@ spec:css-syntax-3;
   <pre class="idl">
     dictionary CredentialRequestOptions {
       CredentialMediationRequirement mediation = "optional";
+      CredentialUiModeRequirement uiMode = "active";
```

Comment:
This should be a DOMString [for these reasons](https://w3c.github.io/webauthn/#sct-domstring-backwards-compatibility).